addons/website_sale/static/src/snippets/s_products_searchbar/000.js

```diff
@@ -74,6 +74,12 @@ publicWidget.registry.productsSearchBar = publicWidget.Widget.extend({
         if (this.displayDescription) {
             res.products.forEach(p => {p.description_sale = Markup(p.description_sale);});
         }
+        if (this.displayPrice) {
```

Comment:
Probably better to do it in the same loop above to not loop twice the products but I guess it is not that big of a deal

Reply:
all depend of the configuration...

If you have !display_price && !displayDescription, you will loop for nothing
So not sure than one better than the other... especially that description is not shown by default ...

Reply:
but yes, with double condition, less readable, but more powerful ;)